backend/models/user.model.ts

```diff
@@ -28,6 +32,23 @@ const UserSchema: Schema = new Schema({
     required: true,
     enum: ["Volunteer", "Staff", "Admin"],
   },
+  startDate: {
+    type: Date,
+    required: true,
+  },
+  birthday: {
+    type: Date,
+    required: true,
+  },
+  locationType: {
+    type: String,
+    required: true,
+    enum: ["Remote", "Hybrid", "In Person"],
```

Comment:
In the types file, for the Location Type enum field you have "InPerson" (no space) v.s "In Person" here.